wifiphisher/common/accesspoint.py

```diff
@@ -151,3 +169,32 @@ def on_exit(self):
         # sleep 2 seconds to wait all the hostapd process is
         # killed
         time.sleep(2)
+
+    def read_connected_victims_file(self):
+        """
+        Reads dnsmasq.leases file and creates and inserts Victim
+        objects to the dictionary
+        """
+
+        if os.path.isfile('/var/lib/misc/dnsmasq.leases'):
+            with open("/var/lib/misc/dnsmasq.leases", "r") as dnsmasq_leases:
+                for line in dnsmasq_leases:
+                    line = line.split()
+                    if len(line) > 0:
```

Comment:
Instead of having to indent for every `if` (which makes the code harder to read), a trick here is to check for the `if not` first and return if needed. For example, here it would be:
```python
if not len(line) > 0:
    return